src/ObjectUtils/PdbFileTest.cpp

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include <filesystem>
+
+#include "ObjectUtils/PdbFile.h"
+#include "OrbitBase/ExecutablePath.h"
+#include "OrbitBase/Logging.h"
+#include "OrbitBase/TestUtils.h"
+
+using orbit_base::HasError;
+using orbit_base::HasNoError;
+using orbit_grpc_protos::SymbolInfo;
+using ::testing::ElementsAre;
+
+TEST(PdbFile, LoadDebugSymbols) {
+  std::filesystem::path file_path_pdb =
+      orbit_base::GetExecutableDir() / "testdata" / "ObjectUtilsTests" / "dllmain.pdb";
+
+  ErrorMessageOr<std::unique_ptr<orbit_object_utils::PdbFile>> pdb_file_result =
+      orbit_object_utils::CreatePdbFile(file_path_pdb);
+  ASSERT_THAT(pdb_file_result, HasNoError());
+  std::unique_ptr<orbit_object_utils::PdbFile> pdb_file = std::move(pdb_file_result.value());
+  auto symbols_result = pdb_file->LoadDebugSymbols();
+  ASSERT_THAT(symbols_result, HasNoError());
+
+  auto symbols = std::move(symbols_result.value());
+
+  std::vector<SymbolInfo> symbol_infos(symbols.symbol_infos().begin(),
+                                       symbols.symbol_infos().end());
+  EXPECT_EQ(symbol_infos.size(), 4982);
+
+  SymbolInfo symbol = symbol_infos[0];
+  EXPECT_EQ(symbol.name(), "PrintHelloWorldInternal");
```

Comment:
interesting, I would expect this name to be mangled... does it mean pbd info always returns detangled names?

Reply:
All other names I get area also demangled, not sure if this is a PDB property or if llvm demangles the name when processing the symbol records (couldn't find anything on this in the code, but could have easily missed it).

Reply:
Add a test for symbol.demangled_name anyways?

Reply:
Yes, of course, done.